### ISSUE TITLE
Joined(Iterable<? extends T> items, T item)

### ISSUE THREAD

**Issue by yegor256** [yegor256/cactoos#1619]
How about we introduce a new ctor `Joined(Iterable<? extends T> items, T item)`, which will add the `item` to the end of the list. Currently, the existing ctor `Joined(T item, Iterable<? extends T> items)` prepends the list with a new item. I didn't manage to find how to append it.

**Comment by victornoel** [comment]
@yegor256 to append it you can use something like: `new Joined(items, new IterableOf(item))` but I agree we can indeed add a constructor for this use case.
I would generify it by proposing we introduce `Joined(Iterable<? extends T> items, T... appended)`

**Comment by victornoel** [comment]
@0crat in

**Comment by victornoel** [comment]
@0crat in

**Comment by victornoel** [comment]
@0crat in

**Comment by william7b7d** [comment]
I can pick this one up.  Seems like a good first contribution.  Do you all have contributor guidelines anywhere?  Looks like there's a `qulice` profile that will run a code style check.

**Comment by victornoel** [comment]
@william7b7d yep, you can make a PR and we will review it. qulice will be executed automatically. If you have doubts about what needs to be done, don't hesitate to ask here first!

**Comment by william7b7d** [comment]
Okay got a PR together.  Let me know if I missed anything.

https://github.com/yegor256/cactoos/pull/1705
